src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
@buinsky 
Actually I started without `.trimmed()`, but `request().post["skip_checking"]`, or any string parameter would contain an extra trailing newline.

That is why `savepath` and `category` are also trimmed.
```
savepath = savepath.trimmed();
category = category.trimmed();
```

The reason roots in `RequestParser::parseFormData()`, I think there is a bug here. Consider the same form data transmitted in `application/x-www-form-urlencoded` and `multipart/form-data`.

```
POST /command/download HTTP/1.1
Host: localhost
Content-Length: 10
Content-Type: application/x-www-form-urlencoded

test=value&test1=value1
```

```
POST /command/download HTTP/1.1
Host: localhost
Content-Length: 140
Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryBSjrpxVB7qhSB9iR

------WebKitFormBoundaryBSjrpxVB7qhSB9iR
Content-Disposition: form-data; name="test"

value
------WebKitFormBoundaryBSjrpxVB7qhSB9iR
Content-Disposition: form-data; name="test1"

value1
------WebKitFormBoundaryBSjrpxVB7qhSB9iR--
```

`request().post["test"]` would be `"value"` in the url-encoded case, and `"value\n"` in the multi-part case.

But then, `RequestParser::parseContent()` is another issue.